### DISCUSSION TITLE
CVE Integration

### DISCUSSION THREAD

**Discussion by gti8sm** [amidaware/tacticalrmm#1755] [General]
Hello,

**Is your feature request related to a problem? Please describe.**
It would be interesting to have a CVE list on TRMM and an alert if the hardware or software inventoried is impacted by a CVE.

**Describe the solution you'd like**
Do you think it is possible to : 
- request the CVE listing and add it on TRMM
- and link software and hardware inventory with this one and configure alerts etc...

**Describe alternatives you've considered**
For the moment i try to had this on my website with TRMM API and CVE Rss, when a keyword match, i genere an alert

Thanks 
have a nice weekend ;)
Simon

**Comment by dinger1986** [comment]
Please follow the template or this will be removed.

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Add any other context or screenshots about the feature request here.

**Comment by gti8sm** [comment]
Sorry the template is not really adpated because is not really a problem !

**Comment by dinger1986** [comment]
I think this could be achieved by a script, can you give more details about how you have implemented your system?

**Comment by P6g9YHK6** [comment]
I have done this via a script and it works fine.
a heavily modified version of this:
https://github.com/vulmon/Vulmap

for any other cve analysis i use nessus.

**Comment by ghost** [comment]
just use Wazuh, it has all the things you said/want. deploy it through tacticalRMM, it can be done in seconds :)

**Comment by badsmoke** [comment]
@P6g9YHK6  vulmap looks good, but is unfortunately very old the last commit is 3 years old, how did you integrate it into trmm?

**Comment by P6g9YHK6** [comment]
> @P6g9YHK6 vulmap looks good, but is unfortunately very old the last commit is 3 years old, how did you integrate it into trmm?

https://github.com/amidaware/community-scripts/blob/main/scripts_wip/Win_Security_Vuln_scanner.ps1

;)

**Comment by silversword411** [comment]
CVE/security assessement is a whole product category unto itself, and will probably never be native to TRMM.

Extensions, scripts etc are the best option for that.
